samples/bluetooth/mesh/CMakeLists.txt

```diff
@@ -12,8 +12,6 @@ project(mesh)
 
 target_sources(app PRIVATE src/main.c)
 
-if (BOARD STREQUAL bbc_microbit)
+if(CONFIG_MICROBIT_DISPLAY)
   target_sources(app PRIVATE src/microbit.c)
-else()
-  target_sources(app PRIVATE src/board.c)
```

Comment:
Is board.c a file that is not used anymore? Shouldn't it be removed in that case, or is this change i mistake?